src/index.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import Categories from './components/Categories';
+import Books from './components/Books';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="/" element={<Books />} />
+        <Route path="Categories" element={<Categories />} />
```

Comment:
App runs but does not show any page. There is a blank page showing on my system. Kindly make sure all the pages of the app are working.